backend/apid/graphql/entity.go

```diff
@@ -94,7 +94,7 @@ func (r *entityImpl) Related(p schema.EntityRelatedFieldResolverParams) (interfa
 }
 
 // Status implements response to request for 'status' field.
-func (r *entityImpl) Status(p graphql.ResolveParams) (int, error) {
+func (r *entityImpl) Status(p graphql.ResolveParams) (interface{}, error) {
```

Comment:
Why not one of the `uint` types instead of `interface{}`?

Reply:
While I agree this would be much better, this method is implementing the [`EntityStateFieldResolver` interface](https://github.com/sensu/sensu-go/pull/2516/files#diff-15797ce5fbb9917807da2d97768c8266R80) found in the `graphql/schema` package. A current limitation of the code generator is that it does not have any way of resolving custom defined types and as such falls back to an `interface{}` when generating it's field resolver interfaces. In the future it would be nice to add support for mapping GraphQL types to Go types / primitives, but I think it is out of scope for this change.

Reply:
Thanks for the clarification regarding the code generator!